### PR TITLE
Modified Flows

### PR DESCRIPTION
**Pull request by hsmurali**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/atacseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/atacseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by hsmurali** [comment]
I have updated the workflow to
1. Interface workflow output to make data for plots
2. Made registry tables to be pulled into plots
3. Added documentation for the workflow

Could you please leave comments to this commit, [31a1cbe](https://github.com/latchbio-nfcore/atacseq/pull/1/commits/31a1cbe2b558e955583aba308d38184771c126a2)?

File: latch_metadata/__init__.py [hunk 1]
```diff
@@ -1,15 +1,145 @@
...
+            "Misc. Sample Information",
+            Params("fragment_size", "seq_center", "with_control"),
+        ),
+    ),
+]
 NextflowMetadata(
-    display_name='nf-core/atacseq',
+    display_name="nf-core/atacseq",
     author=LatchAuthor(
         name="Your Name",
```
Comment (tahirdme) [inline latch_metadata/__init__.py]: change to "nf-core" - we have a cool new graphic

File: latch_metadata/__init__.py [hunk 2]
```diff
@@ -1,15 +1,145 @@
...
 
+flow = [
+    Section("Input/Output", Params("input", "outdir", "run_name")),
+    Section(
+        "Reference Genome Options",
+        Fork(
+            "genome_source",
+            "",
+            latch_genome_source=ForkBranch(
+                "Latch Certified Reference Genome", Params("genome")
```
Comment (tahirdme) [inline latch_metadata/__init__.py]: You should call this latch_genome

We should still allow people to use iGenomes if they want honestly 
So call this "latch_genome" and leave a "genome" in the Hidden Options for optional iGenomes - might be handy for us in a fix

File: wf/entrypoint.py [hunk 3]
```diff
@@ -1,29 +1,360 @@
...
+@custom_task(cpu=4, memory=48, storage_gib=50)
+def Run_Rscript(map_input: InputMap_ATACQC) -> LatchDir:
+    run_flag = map_input.run_flag
+    sample = map_input.sample
+    bamfile = map_input.bamfile
+    baifile = map_input.baifile
+    outdir = map_input.outdir
+
+    local_dir = sample + "/"
+    shutil.copy(bamfile.local_path, str(Path().resolve()))
```
Comment (tahirdme) [inline wf/entrypoint.py]: Confused about the .copy() here which is followed by the .download() a few lines below

File: wf/entrypoint.py [hunk 4]
```diff
@@ -1,29 +1,360 @@
...
+@custom_task(cpu=4, memory=48, storage_gib=50)
+def Merge_pq_files(
+    cov_parquet_path: typing.List[LatchDir], outPath: LatchDir
+) -> LatchDir:
+    chromosomes = {}
+    for cov in cov_parquet_path:
+        files = latch_listdir(cov)
+        for s in files:
+            print(s)
+            splits = Path(s).name.split(".")
```
Comment (tahirdme) [inline wf/entrypoint.py]: Can you confirm that wrapping a LPath().path link in a Path object won't download it

File: wf/entrypoint.py [hunk 5]
```diff
@@ -1,29 +1,360 @@
-from dataclasses import dataclass
```
Comment (tahirdme) [inline wf/entrypoint.py:1]: Can you add function level descriptions to your code?

Hard to figure out what each function is supposed to be doing 

With this under the def()
"""
exampleexampleexampleexample
""""

Comment (tahirdme) [reply wf/entrypoint.py:1]: Would also recommend splitting up the nextflow entrypoint from the Registry and postprocessing code in separate .py files

Makes it easier to read and understand

**Review by tahirdme — CHANGES_REQUESTED** [review]

**Review by hsmurali — COMMENTED** [review]
@tahirdme I have tested the launch plans, it works fine and I also added the igenomes back. I have also changed the s3 links with the test dataset.

**Pull request merged**